python/konduit/client.py

```diff
@@ -10,7 +10,31 @@
 
 class Client(object):
     def __init__(self, timeout=60, input_type='NUMPY', endpoint_output_type='NUMPY',
-                 return_output_type=None, input_names=['default'], output_names=['default'], url=None):
+                 return_output_type=None, url=None):
+
+        try:
+            r = requests.get("{}/healthcheck".format(url))
+            if r.status_code != 204:
```

Comment:
I was thinking, maybe we can make this `try` block a utility function to be used in other places. because if we don't have that, how would users verify that the server is running?

Reply:
Ah that's a good idea for sure.

Reply:
Created a utility for that. The server validation is a little different from client so it wasn't created into a utility.